package.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
These dependencies do not belong to this package.json.

Reply:
this again? I am not sure what's happening here why this package.json gets pushed.  will take a look